### PR TITLE
handle web_test webhandler errors.

### PR DESCRIPTION
**Pull request by krasi-georgiev**
will handle errors like: `panic: Can't start webhandler error listen tcp :9090: bind: address already in use`

### CODE REVIEW

File: web/web_test.go [hunk 1]
```diff
@@ -108,7 +108,12 @@ func TestReadyAndHealthy(t *testing.T) {
 	opts.Flags = map[string]string{}
 
 	webHandler := New(nil, opts)
-	go webHandler.Run(context.Background())
+	go func() {
+		err := webHandler.Run(context.Background())
+		if err != nil {
+			panic(fmt.Sprintf("Can't start web handler:%s", err))
```
Comment (Gouthamve) [inline web/web_test.go]: Why is this a panic and not `t.Fatalf/Errorf`?

**Comment by krasi-georgiev** [comment]
@Gouthamve I have no idea , what I was thinking :) fixed now.

**Comment by krasi-georgiev** [comment]
@Gouthamve added a small fix for the currently  failing tests for `main_test.go`

seems some differnce in OS again as locally when a process has exited from `os.Interrupt` it doesn't return any error but in travis is returns `signal: interrupt`.

**Comment by Gouthamve** [comment]
👍

**Pull request merged**